content/source/docs/cloud/agents/index.html.md

```diff
@@ -23,6 +23,14 @@ The agent architecture is pull-based, so no inbound connectivity is required. An
 
 Agents support Terraform versions 0.12 and above. Workspaces configured to use Terraform versions below 0.12 will not be able to select the agent-based execution mode.
 
+### Hardware Requirements
+
+The host running the agent will have varying resource requirements depending on the workspace being run.  A host for an agent can be a dedicated or shared cloud instance, virtual machine, bare metal server, or a container.  Memory, CPU and disk space should be monitored and adjusted accordingly as usage and performance will vary upon the workspaces being executed. The specs below are provided as a reference. Depending on the type of environment you deploy to, the name of your instance type may vary.
```

Comment:
Just thinking about active voice, flow (I want the specs sentence to come right before the specs), and concision. Please make sure I didn't materially change the meaning!

```suggestion
The host running the agent will have varying resource requirements depending on the workspace. A host can be a dedicated or shared cloud instance, virtual machine, bare metal server, or a container. You should monitor and adjust memory, CPU, and disk space based on each workspace's usage and performance. The name of your instance type may vary depending on your deployment environment. 

The specs below are provided as a reference. 
```